File: sdkwork-app-sdk-go/types/user_coupon_vo.go
package types

// 用户优惠券响应
type UserCouponVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	UserCouponId string `json:"userCouponId"`
	CouponId string `json:"couponId"`
	CouponName string `json:"couponName"`
	CouponCode string `json:"couponCode"`
	Type string `json:"type"`
	TypeName string `json:"typeName"`
	Amount int `json:"amount"`
	Discount float64 `json:"discount"`
	MinConsume int `json:"minConsume"`
	AcquireAt string `json:"acquireAt"`
	AcquireRequestNo string `json:"acquireRequestNo"`
	AcquireType string `json:"acquireType"`
	PointCost int `json:"pointCost"`
	PointsRefunded bool `json:"pointsRefunded"`
	PointsRefundAt string `json:"pointsRefundAt"`
	UseAt string `json:"useAt"`
	ExpireAt string `json:"expireAt"`
	Status string `json:"status"`
	StatusName string `json:"statusName"`
	OrderId string `json:"orderId"`
	CanShared bool `json:"canShared"`
	Available bool `json:"available"`
	RemainingDays int `json:"remainingDays"`
}

File: sdkwork-app-sdk-csharp/Models/UserCouponVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class UserCouponVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? UserCouponId { get; set; }
        public string? CouponId { get; set; }
        public string? CouponName { get; set; }
        public string? CouponCode { get; set; }
        public string? Type { get; set; }
        public string? TypeName { get; set; }
        public int? Amount { get; set; }
        public double? Discount { get; set; }
        public int? MinConsume { get; set; }
        public string? AcquireAt { get; set; }
        public string? AcquireRequestNo { get; set; }
        public string? AcquireType { get; set; }
        public int? PointCost { get; set; }
        public bool? PointsRefunded { get; set; }
        public string? PointsRefundAt { get; set; }
        public string? UseAt { get; set; }
        public string? ExpireAt { get; set; }
        public string? Status { get; set; }
        public string? StatusName { get; set; }
        public string? OrderId { get; set; }
        public bool? CanShared { get; set; }
        public bool? Available { get; set; }
        public int? RemainingDays { get; set; }
    }
}
